src/makeCalculator.js

```diff
@@ -4,7 +4,50 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(num) {
+      this.result += num;
+
+      return this;
+    },
+
+    subtract(num) {
+      this.result -= num;
+
+      return this;
+    },
+
+    multiply(num) {
+      this.result *= num;
+
+      return this;
+    },
+
+    divide(num) {
+      if (num !== 0) {
+        this.result /= num;
+      } else {
+        return 'Error: Division by zero';
+      }
```

Comment:
In the `divide` method, when division by zero is attempted, an error message is returned instead of the calculator object. This breaks the chainability of the calculator methods. Instead, you could consider throwing an Error, which would immediately halt execution and would be more appropriate for such an exceptional case. Here is how you could do it:

javascript
if (num === 0) {
  throw new Error('Error: Division by zero');
}
```

This way, if a division by zero is attempted, an error will be thrown, and the rest of the chain will not be executed.